test_suite/README.md

```diff
@@ -201,19 +187,14 @@ def peripheral(board_allocator: BoardAllocator):
     board_allocator.release(device)
 ```
 
-The decorator declares a fixture with a function scope meaning the object returned
-is destroyed after the function that used it ends. 
-Then the function prototype accepts a `board_allocator` argument. It is automatically
-resolved by pytest to the `board_allocator` fixture. 
-Then a board is allocated and the BLE API is initialised.
-
-The device object is then returned to the pytest framework using the `yield` keyword. 
-
-The function is reentered at the end of the fixture scope and the board is released. 
-
+1. The decorator declares a fixture with a function scope, meaning the returned object is destroyed after the function that used it ends.
+1. The function prototype accepts a `board_allocator` argument, which pytest automatically resolves to the `board_allocator` fixture.
+1. A board is allocated and the BLE API is initialised.<!--who does the allocating?-->
+1. The device object is returned to the pytest framework using the `yield` keyword.<!--who does the returning?-->
```

Comment:
The `peripheral` fixture returns the `device` object.

Reply:
```suggestion
1. The peripheral fixture uses the `yield` keyword to return the device object to the pytest framework.
```